app/controllers/testOnly/TestOnlyRouterController.scala

```diff
@@ -43,12 +43,27 @@ class TestOnlyRouterController @Inject()(
         .map(response => Status(response.status))
   }
 
-  def toCoreMessage: Action[NodeSeq] = action.async(parse.xml) {
+  def arrivalNotificationMessageToCore: Action[NodeSeq] = action.async(parse.xml) {
     implicit request =>
-      Log.debug(s"To Core Request Body (Controller): ${request.body}")
-      Log.debug(s"To Core Request Headers (Controller): ${request.headers}")
+      Log.debug(s"Arrival Notification To Core Request Body (Controller): ${request.body}")
+      Log.debug(s"Arrival Notification To Core Request Headers (Controller): ${request.headers}")
       connector
-        .submitOutboundMessage(request.body, request.headers)
+        .createArrivalNotificationMessage(request.body, request.headers)
         .map(response => Status(response.status))
   }
+
+  def unloadingRemarksMessageToCore: Action[NodeSeq] = action.async(parse.xml) {
```

Comment:
This actually takes a generic message so the messages are not necessarily an Unloading Remark